cmd/protoc-gen-go-grpc/grpc.go

```diff
@@ -515,6 +517,25 @@ func genLeadingComments(g *protogen.GeneratedFile, loc protoreflect.SourceLocati
 	}
 }
 
+func genFullMethods(g *protogen.GeneratedFile, service *protogen.Service) {
+	g.P("const (")
+	cNames := make([]string, len(service.Methods))
+	for i, method := range service.Methods {
+		cName := fmt.Sprintf("%s_%s_FullMethod", service.GoName, method.GoName)
```

Comment:
@dfawley 

Following up on our offline discussion where we wanted to emphasize on the fact that the newly generated symbols should have very little chance of a name collision with any existing names.

The current approach taken here is: `<ServiceName>_<MethodName>_FullMethod`

What do you think about: `Service_<ServiceName>_Method_<MethodName>_FullMethodName`? This is verbose, but would have less chance of name collision.

Reply:
I think the likelihood of a conflict is sufficiently small with this approach.  But how about `<ServiceName>_<MethodName>_FullMethodName` to be more explicit that the symbol represents a method name string (as opposed to a method handler function/interface type, e.g.)?

Reply:
Happy to make the change, are we in agreement that the second proposal is enough to avoid name conflicts? (i.e. `<ServiceName>_<MethodName>_FullMethodName`)

Reply:
Latest push has the second option, also moved the string formatting to a helper function in the interface